### ISSUE TITLE
Scan values into dbMap.Get() argument

### ISSUE THREAD

**Issue by keepzero** [go-gorp/gorp#136]
In `func (*DbMap) Select` values are returned in one of two ways.
Why `func (*DbMap) Get` not write result into i if i is a pointer to a struct?
And if `Get` use the key value contained in i, user will not need to care about the order of key column specified in `SetKeys()`.
Is it possible?

**Comment by GeertJohan** [comment]
Yes this could be a good addition. I'm setting this for v2.1-maybe, please see #270
